### ISSUE TITLE
Quest redo-timer resets when a player relogs

### ISSUE THREAD

**Issue by gtwy** [FlyingPikachu/Quests#185]
If a redo-timer is set, all a player has to do is log out and log back in and they can immediately do the quest again. This is especially bad for quests with a high reward that should only be allowed once per week or once per month.

Sorry to submit so many of these bugs, but I plan on heavily utilizing this plugin on our server and I am finding a lot of issues with it as I go.

**Comment by AlphabetoPT** [comment]
My Quests Plugin is disabled because of this.
Waiting on a reply.
Thanks

**Comment by bartonsweb** [comment]
I am also having this issue, I cannot enable the plugin until fixed because players will be able to abuse the workaround of the redo-timer and farm the quests.

Any update on if this will be fixed?
Thanks

**Comment by BlackBeltPanda** [comment]
I'm also having this issue; wouldn't have noticed if you hadn't reported it. Looks like I'll have to wipe all the points my players earned. -_-*

**Comment by FlyingPikachu** [comment]
Duplicate of #67
